scripts/install_onnx_gpu_for_amd.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-# This script is meant to be run at buildtime. This is because onnxruntime-gpu
-# doesn't install on arm64
-if ! [[ "$TARGETPLATFORM" ]]; then
-  echo ERROR environment var TARGETPLATFORM is undefined
-  exit 1
-fi
-
-if [[ "$TARGETPLATFORM" != "linux/arm64" ]]; then
-  pip3 --no-cache-dir install --upgrade onnxruntime-gpu==1.12.1
-else
-  pip3 --no-cache-dir install --upgrade onnxruntime
```

Comment:
does this always install the latest version?

Reply:
The version in 2.13.1 is 2.19.1 and I think this is a mistake. We are using `onnx==1.12.0` and it is only compatible with onnxruntime  2.13 and 1.12